.rubocop.yml

```diff
@@ -20,6 +20,34 @@ Metrics/BlockLength:
 Style/NumericLiteralPrefix:
   EnforcedOctalStyle: zero_only
 
+# Enable new cops
```

Comment:
We can use a new style here called
```
AllCops:
  NewCops: enable
```

Nested underneath the above.

Reply:
For what is worth, I prefer explicitly enabling cops that we're interested in (even if it's all of them), rather than blindly accepting every new from rubocop.